src/librustdoc/html/render/print_item.rs

```diff
@@ -1892,7 +1892,7 @@ fn document_non_exhaustive(w: &mut Buffer, item: &clean::Item) {
 }
 
 fn document_type_layout(w: &mut Buffer, cx: &Context<'_>, ty_def_id: DefId) {
-    fn write_size_of_layout(w: &mut Buffer, layout: Layout<'_>, tag_size: u64) {
+    fn write_size_of_layout(w: &mut Buffer, layout: &LayoutS<VariantIdx>, tag_size: u64) {
```

Comment:
Why is this rustdoc change needed?

Reply:
The layout of variants nested inside a layout are now not interned because rustc's interning is not available outside of rustc. And rustdoc calls this function on those nested layouts, so I changed it.